### DISCUSSION TITLE
Regarding implementation of AODV protocol without inet.

### DISCUSSION THREAD

**Discussion by masud0932** [omnetpp/omnetpp#1086] [General]
Hi, my supervisor asked me to implent the aodv protocol for manet in omnet++ without using inet framework just to check my skill before assigning me for a project. Particularly, he told me to write the cc file for measuring the queue length of each node. So, could you please help me writing source codes or giving me any idea how to start. Thanks !!!

**Comment by rhornig** [comment]
How about doing the tictoc tutorial as a starter: https://docs.omnetpp.org/tutorials/tictoc/part5/
